src/utils/motebehovUtils.ts

```diff
@@ -179,6 +185,60 @@ export const toMotebehovTilbakemeldingDTO = (
   };
 };
 
-export function isSykmeldtSvar(motebehov) {
+export function isArbeidstakerMotebehov(motebehov: MotebehovVeilederDTO) {
   return motebehov.opprettetAv === motebehov.aktorId;
 }
+
+export function mapMotebehovToMeldtMotebehovFormat(
+  motebehov: MotebehovVeilederDTO[]
+): MeldtMotebehov[] {
+  return motebehov
+    .filter(
+      (motebehov) => motebehov.skjemaType === MotebehovSkjemaType.MELD_BEHOV
+    )
+    .map((motebehov) => {
+      return {
+        id: motebehov.id,
+        opprettetDato: motebehov.opprettetDato,
+        opprettetAv: motebehov.opprettetAv,
+        opprettetAvNavn: motebehov.opprettetAvNavn,
+        innmelder: isArbeidstakerMotebehov(motebehov)
+          ? MotebehovInnmelder.ARBEIDSTAKER
+          : MotebehovInnmelder.ARBEIDSGIVER,
+        arbeidstakerFnr: motebehov.arbeidstakerFnr,
+        virksomhetsnummer: motebehov.virksomhetsnummer,
+        begrunnelse: motebehov.motebehovSvar.forklaring,
```

Comment:
I `MELD_BEHOV` er det ikke noe vits i å ha feltet `motebehovSvar.harMotebehov` ettersom et innmeldt behov jo er et møtebehov av natur 🤷🏼‍♂️ Mens for `SVAR_BEHOV` gir det mening, da AT eller AG kan mene at de ikke trenger møte når vi spør.